### DISCUSSION TITLE
Allow user preference for saving settings

### DISCUSSION THREAD

**Discussion by edhans** [TabularEditor/TabularEditor3#1054] [Ideas]
I'd like a setting to allow Tabular Editor 3 settings to be stored in a user selectable folder. I want my settings (Preferences.json, MacroActions.json, etc.) stored in a OneDrive location so they are both backed up and can be easily sync'd between multiple PCs.

Or a cloud account to store these settings where logging in retrieves them, but I suspect a local folder sync'd via OneDrive would be easier.

**Comment by JamesDBartlett3** [comment]
I'm migrating to a new machine this week. Has this been implemented yet?
